pallets/amm/src/lib.rs

```diff
@@ -107,24 +107,23 @@ pub mod pallet {
 	#[pallet::event]
 	#[pallet::generate_deposit(pub(crate) fn deposit_event)]
 	pub enum Event<T: Config> {
-		/// AddLiquidity
-		/// who, asset_a, asset_b, amount_a, amount_b
-		AddLiquidity(T::AccountId, AssetId, AssetId, Balance, Balance),
+		/// New liquidity was provided to the pool. [who, asset_a, asset_b, amount_a, amount_b]
+		LiquidityAdded(T::AccountId, AssetId, AssetId, Balance, Balance),
```

Comment:
Personally, I like the format of `AddedLiquidity`, `RemovedLiquidity` etc.

But it is my personal preference. But again, let's settle on one format ( because I already made changes in my new pallet with AddedToken, AddedLiquidity etc). 

Happy to change my mind of course. 

What do you think ?

Reply:
Hey @enthusiastmartin, thanks for checking it out. My instinct is the `LiquidityAdded` is better. In `AddedLiquidity` Added is the adjective describing Liquidity. With that convention eg. UserRegistered – would be: RegisteredUser.

I would be happy to keep subject + verb in past-tense.

Reply:
I think Noun (subject) -> Verb (action) is more standard convention. 

Even better might be group -> subject -> action 
That way, you can instantly see what is the part of code you are working with and then what is happening with it. If you reverse this you'll first see verbs and just then if it's anything you are interested in.

Reply:
I'd have a look at substrate conventions and if there is no hard one. Go with one and add it to our contributing guidelines.

Reply:
Ok. fine with me. I'll change mines to this form too.